view/form/DeliveryForm.php

```diff
@@ -107,7 +107,16 @@ protected function setThemeNameSelectorOptions()
             }
         }
 
-        $this->form->getElement($elementUri)->setOptions($options);
+        $themeElement = $this->form->getElement($elementUri);
+
+        $themeElement->setOptions($options);
+
+        // Ensures stored value is selected in the element
+        $themeElement->setValue(
+            $this->instance->getOnePropertyValue(
+                $this->instance->getProperty(DeliveryThemeDetailsProvider::DELIVERY_THEME_ID_URI)
+            )
+        );
```

Comment:
I might be wrong, but we need to check some details.

- Is this a backport? Cause I do not see this code on `develop` or `master` branches.
- The `$this->instance` can be null, so we need to validate this scenario too.

Reply:
@gabrielfs7, this is a backport. In fact, it was only caused due to another bad backport. I was considering targeting `develop` branch but it's not reproducible in other later versions.

For the second point, I have just updated it. :)

Reply:
Ok got it, my impression was that we were going to backport the extension adding the missing endpoint instead. Wont this missing extension backport "keep" other issues?

Thanks for the update

Reply:
I updated the version rather than creating more backports to avoid such issues. ;)

https://github.com/oat-sa/tao-community/pull/340/files